src/components/Alert/Alert.jsx

```diff
@@ -1,26 +1,41 @@
+import { useEffect } from "react";
+import { useAlert } from "../../context/alert-context";
 import "./alert.css";
 
-function Alert({ type, msg }) {
+function Alert() {
+	const { alertState, alertDispatch } = useAlert();
 	let alertClass = "";
 	let spanClass = "info";
 
-	if (type === "success") {
+	if (alertState.alertType === "success") {
```

Comment:
instead of doing alertState.alertType everywhere destruct it once used it accordingly

Reply:
Okay, thank you